modules/utils.js

```diff
@@ -94,7 +94,8 @@ function betterInstanceOf(input, constructor) {
       if (typeof inputToTest == 'undefined') {
         return false;
       }
-      inputToTest = inputToTest.__proto__;
```

Comment:
This change breaks the SlimerJS test. This kind of regression is exactly why I proposed to
- make lint changes *only* (no functional change whatsoever)
- make *one PR per linted file*

here: https://github.com/n1k0/casperjs/issues/1430#issuecomment-180319741.

Some of the problems could be solved by adjusting the .eslintrc -- e.g. having strictly quoted keys in Objects is not a good thing to have; so I would personally still prefer
```js
        type: "assertUrlMatch",
        standard: "Current url matches the provided pattern",
        values: {
            currentUrl: currentUrl,
            pattern: pattern.toString()
```
over
```js
        "type": "assertUrlMatch",
        "standard": "Current url matches the provided pattern",
        "values": {
            "currentUrl": currentUrl,
            "pattern": pattern.toString()
```